site/content/guides/global-rate-limiting.md

```diff
@@ -424,12 +424,12 @@ For more information, see the [Contour rate limiting documentation][7] and the [
 The YAML used in this guide is available [in the Contour repository][9].
 
 [1]: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_features/global_rate_limiting
-[2]: /docs/{{site.latest}}/deploy-options/#kind
+[2]: /docs/{{< param docs_latest >}}/deploy-options/#kind
 [3]: https://projectcontour.io/getting-started/#option-1-quickstart
 [4]: https://github.com/envoyproxy/ratelimit
 [5]: https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ratelimit/v3/rls.proto
 [6]: https://github.com/envoyproxy/ratelimit#configuration
-[7]: /docs/{{site.latest}}/config/rate-limiting/
-[8]: /docs/{{site.latest}}/config/api/
-[9]: {{site.github.repository_url}}/tree/main/examples/ratelimit
+[7]: /docs/{{< param docs_latest >}}/config/rate-limiting/
+[8]: /docs/{{< param docs_latest >}}/config/api/
+[9]: {{< param github_base_url>}}/tree/main/examples/ratelimit
```

Comment:
```suggestion
[9]: {{< param github_url >}}/tree/main/examples/ratelimit
```

Reply:
We had two for some reason, took out the extra.

Reply:
Yeah just noticed that, 👍